tests/js/ESLintBearTest.py

```diff
@@ -67,3 +77,36 @@
     create_tempfile=False,
     settings={'eslint_config': os.path.join(test_dir,
                                             'eslintconfig_import.json')})
+
+
+@generate_skip_decorator(ESLintBear)
+class ESLintBearIgnoredFileTest(LocalBearTestHelper):
+
+    def setUp(self):
+        self.section = Section('')
+        self.uut = ESLintBear(self.section, Queue())
+
+    def test_lint_config_file(self):
+
+        self.maxDiff = None
+        config_filename = os.path.join(test_dir, '.eslintrc.js')
+
+        self.section.append(Setting('eslint_config', config_filename))
+
+        expected = Result.from_values(
+            'ESLintBear',
+            'File ignored by default.  Use a negated ignore pattern '
```

Comment:
Note ive created https://github.com/coala/coala-bears/issues/2607 to determine how we handle this properly.